modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/reader/BatchEngineImportTaskItemReaderUtil.java

```diff
@@ -60,13 +62,29 @@ public static <T> T convertValue(
 				}
 			}
 
+			JsonDeserialize[] jsonDeserialize = field.getAnnotationsByType(
```

Comment:
The name does not match the Brian's pattern. JsonDeserialize is an array, not a unique element, so I guess the name should be something like `jsonDeserializeArray` or `jsonDeserializes`